File: JavaScript/1018 - Cédulas.js
var N = parseInt(require('fs').readFileSync('/dev/stdin', 'utf8'));
console.log(N);

var notas = [100, 50, 20, 10, 5, 2, 1];
for (let nota of notas) {
    let qtd = Math.floor(N / nota);
    console.log(`${qtd} nota(s) de R$ ${nota},00`);
    N %= nota;
}

File: Clojure/1018 - Cédulas.clj
(let [valor (Integer/parseInt (read-line))
      notas [100 50 20 10 5 2 1]]
  (println valor)
  (loop [v valor n notas]
    (when (seq n)
      (let [nota (first n)
            quantidade (quot v nota)]
        (println (format "%d nota(s) de R$ %d,00" quantidade nota))
        (recur (mod v nota) (rest n))))))